domain/src/main/java/org/commcare/dalvik/domain/model/AbhaDetailModel.kt

```diff
@@ -51,7 +52,10 @@ class AbhaDetailModel() {
                 }
                 aadhaarDataList.add(
                     KeyValueModel(
-                        LanguageManager.getTranslatedValue(key),
+                        abhaHealthLangKeysMap[key]?.let {
```

Comment:
When will the `key` not be present in the `abhaHealthLangKeysMap` ? If it will only happen due to a code bug, It's probably better to throw an error here when key is not present in the map. That will allow us to spot such errors faster in future.

Reply:
"abhaHealthLangKeysMap" translation keys are only used for the screen when the ABHA number is generated and the AADHAAR data is being shown to user.

For now we have mapped all the key coming in the AADHAR data JSON  from the ABDM server. In cases a new key arrives in JSON then we need to add that to the hash map and also update the values in string.xml files.

In case any key translation is missing we just show the key as fallback in the UI.